packages/langium/src/grammar/type-system/type-validator.ts

```diff
@@ -25,7 +25,7 @@ export function validateTypesConsistency(grammar: Grammar, accept: ValidationAcc
     }
 
     // Report missing assignments for required properties in offending nodes
-    function applyMissingAssignmentErrorToRuleNodes(nodes: readonly ParserRule[], typeName: string, accept: ValidationAcceptor): (propertyName: string, errorMessage: string) => void {
+    function applyMissingAssignmentErrorToRuleNodes(nodes: readonly ParserRule[], typeName: string): (propertyName: string, errorMessage: string) => void {
```

Comment:
Ah, good catch! I had this function defined outside of `validateTypesConsistency` to start with, and forgot to remove the redundant param once it was back in.